code/modules/shuttle/navigation_computer.dm

```diff
@@ -8,8 +8,10 @@
 	var/shuttlePortId = ""
 	var/shuttlePortName = ""
 	var/list/jumpto_ports = list() //hashset of ports to jump to and ignore for collision purposes
-	var/list/blacklisted_turfs
-	var/obj/docking_port/stationary/my_port
+	var/list/blacklisted_turfs //turfs we cannot go on, by default, any turf covered by a docking port that we did not place and cannot jump to
+	var/obj/docking_port/stationary/my_port //the custom docking port placed by this console
+	var/obj/docking_port/mobile/shuttle_port //the mobile docking port of the connected shuttle
+	var/list/shuttle_port_areas = list() //hashset of areas owned by shuttle_port
```

Comment:
This list is already stores in mobile docks

Reply:
I did not realize that was a hashset as well.